docs/manifest.md

```diff
@@ -78,6 +78,7 @@ Here is the sample manifest file of the hello world application having two exten
       "en": "Some short description (en)"
     },
     "iconUrl": "https://someurl.com/image.png",
+    "externalInstallationUrl": "https://somestore.com/acme/application",
```

Comment:
I'd put it close to `configuration` section because they are interchangeable. Either "internal" installation with config or "external" installation

Reply:
done